### ISSUE TITLE
Updated Excel Sheet for ASVS!

### ISSUE THREAD

**Issue by deFr0ggy** [OWASP/ASVS#1318]
I just worked onto updating the CSV format specifically for the people who make extensive use of EXCEL. Beautified it a little so its easy for the eyes to keep up. Plus removed the columns which were not required. 

Screenshot and updated file attached!

**Issue 1**

<img width="234" alt="Removed Chapter Name Column" src="https://user-images.githubusercontent.com/16836050/179764570-a6ab66c4-f7ff-4c6e-b81e-dd863e012d17.PNG">

**Issue 2**

<img width="220" alt="Level Issue" src="https://user-images.githubusercontent.com/16836050/179764563-c3d7168c-e8fb-4dac-b9d6-df8726a86515.PNG">

**Updated Excel**

[OWASP.Application.Security.Verification.Standard.4.0.3-en.csv](https://github.com/OWASP/ASVS/files/9141517/OWASP.Application.Security.Verification.Standard.4.0.3-en.csv)

**Updated Excel Screenshot**

<img width="915" alt="Updated Excel" src="https://user-images.githubusercontent.com/16836050/179764793-89549cc5-3036-4a51-81c2-aa145fb25ff5.PNG">

Will work on it a little more later on to build up a dashboard for tracking and reporting! 

Thank you!

**Comment by elarlang** [comment]
As we generate those files automatically, we are not going to change them with manually edited ones.

If you have clear proposals for "Makefile guys", what are the expected changes, then it's maybe automatically generated in the future.

**Comment by tghosth** [comment]
Yeah as Elar said, we tend to restrict our outputs to those which are automatically generated. 

However, if you fancy looking at the python files in [this folder](https://github.com/OWASP/ASVS/tree/master/4.0) and figuring out why we have those weird characters you highlight in "issue 2" above, that would help a lot :)

**Comment by deFr0ggy** [comment]
The issue seems to be with MS Excel as by default it does not detects the Unicode Characters. In order for it to read file correctly, we need to load the file in the following manner. 

DATA -> From Text/CSV -> Change The Origin (1252: Western European to 65001: Unicode 0 UTF-8) -> Save 

Will check if we can have this fixed somehow in python script as well!

**Comment by tghosth** [comment]
It is definitely a python problem as it is already exported like that into the CSV text file

**Comment by deFr0ggy** [comment]
Yes, the file is being exported in UTF-8 format although the default format of MS Excel is not same which causes rendering of UTF-8 characters into other ones!

**Comment by tghosth** [comment]
My sincere apologies, I mis-spoke above. It actually looks correct in a text editor so it is just in Excel where it looks weird so I am open to suggestions if there is a way to export in a text encoding format that will work immediately better in Excel.

**Comment by ike** [comment]
I would suggest we use a converter to xlsx, there are a lot of problems getting Excel data types and unicode support from a CSV without also sacrificing the usability of the CSV in other automated settings.

This is a [small binary converter](https://gitlab.com/DerLinkshaender/csv2xlsx/-/blob/master/tutorial/tutorial.md) we could use in the document builder which supports column data types, unicode, date conversions, etc.

**Comment by tghosth** [comment]
Happy to see a PoC :)
